systemd/systemd.go

```diff
@@ -371,8 +371,9 @@ type Systemd interface {
 	Restart(services []string) error
 	// Reload or restart the service via 'systemctl reload-or-restart'
 	ReloadOrRestart(services []string) error
-	// RestartAll restarts the given service using systemctl restart --all
-	RestartAll(service string) error
+	// RestartSingleCmd restarts the given services using systemctl restart,
```

Comment:
the name in the doc here doesn't match the actual method name

Reply:
Good catch, fixed now